src/util/scriptQueryParameters.spec.ts

```diff
@@ -7,7 +7,9 @@ function createScriptElement(src: string): void {
 }
 
 function removeScriptElement(src: string): void {
-  const scripts: Element[] = [...document.getElementsByTagName("script")];
+  const scripts: Element[] = [].slice.call(
+    document.getElementsByTagName("script")
+  );
```

Comment:
tip: If you can spread, you can achieve same result with `[...document.getElementsByTagName('script')]`

Reply:
Yeah, I would normally use the spread pattern as @clight-fastly suggests. However sadly the compiler doesn't like it. As a NodeList is an Array like object and not a true Array, it seems to think it doesn't have `Symbol.iterator`, which is wrong and clearly a bug in the lib.dom typings. Rather than fight the compiler it was just easier to do this. 

> error TS2488: Type 'HTMLCollectionOf<HTMLScriptElement>' must have a '[Symbol.iterator]()' method that returns an iterator.